### DISCUSSION TITLE
How route rules works?

### DISCUSSION THREAD

**Discussion by Barbapapazes** [nitrojs/nitro#1644] [Q&A]
Hello,

I'm currently playing with route rules and there is some points that I don't understand.

Event Handler on `/`

```ts
export default defineEventHandler(async () => {
  const star = await $fetch('https://ungh.cc/stars/unjs/nitro')

  return {
    star,
    date: new Date().toLocaleString(),
  }
})
```

Nitro Config

```ts
export default defineNitroConfig({
  routeRules: {
    '/': {
      prerender: true,
      static: true,
    }
  }
});
```

Then, I use `npm run build`. But

- Why, even when using prerender, the server send me a fresh response? (Date in the `public/index` file is not the one returned)
- What does the option `static`

Using these option, I was thinking that a static prerendered route will always return the static file, not a fresh response.

**Comment by pi0** [comment]
We might have an exceptional check for `/` route. Does it happens with another route / rule like `/stars` too? (A minimum repo would be nice to help checking it)
